client/app/components/groupInfo/groupInfo.html

```diff
@@ -2,7 +2,7 @@
   <md-toolbar class="md-hue-1">
     <div class="md-toolbar-tools">
       <h1 class="md-title" flex>
-        <md-button ui-sref="groupList" class="md-icon-button">
+        <md-button ui-sref="joinGroupList" class="md-icon-button">
```

Comment:
Sure, we can have a look together if you want. The end-to-end tests spin up a completely new environment (frontend, backend, fresh database) and we can run tests using [protractor](http://www.protractortest.org)

Currently there are only two very basic tests: https://github.com/yunity/foodsaving-e2e/blob/master/e2e.spec.js

They can be easily extended, e.g. making sure that clicking on the button you fixed directs the user to the group list.
Would you be interested to try it out?